java/client/src/test/java/glide/ffi/FfiTest.java

```diff
@@ -88,7 +88,7 @@ public void redisValueToJavaValue_Int(Long input) {
 
     @Test
     public void redisValueToJavaValue_BulkString() {
-        String input = "😀\n💎\n🗿";
+        String input = "dummyString";
```

Comment:
I think we need to use a non-ASCII string there.
Can you try \xHHHH codes?
@jonathanl-bq, what do you think?

Reply:
Yeah, Shachar wanted this to be non-ASCII. I'm looking into it.

Reply:
I updated this to use a non-ASCII UTF-8 sequence from a byte[].